Sources/Authgear.swift

```diff
@@ -338,6 +369,26 @@ public class Authgear: NSObject {
         }
     }
 
+    public func authorizeWithoutSession(
```

Comment:
We should not add this function. Instead, we should add an option to Authgear `prefers SFSafariViewController`, which is false by default. If this option is true, then SFSafariViewController is used instead of ASWebAuthenticationSession.

Reply:
Added